src/vm/errors/vm_errors.rs

```diff
@@ -129,6 +132,9 @@ impl fmt::Display for VirtualMachineError {
             VirtualMachineError::DiffIndexComp(a, b) => {
                 write!(f, "Failed to compare {:?} and  {:?}, cant compare two relocatable values of different segment indexes", a, b)
             },
+            VirtualMachineError::ValueOutside250BitRange(value) => write!(f, "Value: {:?} is outside of the range [0, 2**250)", value),
+            VirtualMachineError::SqrtNegative(value) => write!(f, "Cant calculate the square root of negative or zero number: {:?})", value),
```

Comment:
Otherwise, if we still consider `0` to not have a square root:
```suggestion
            VirtualMachineError::SqrtNegative(value) => write!(f, "Can't calculate the square root of non-positive number: {:?})", value),
```

Reply:
Chose previous solution